minio/api.py

```diff
@@ -434,19 +434,18 @@ def remove_all_bucket_notifications(self, bucket_name):
         """
         is_valid_bucket_name(bucket_name)
 
-        content = xml_marshal_bucket_notifications({})
+        content_bytes = xml_marshal_bucket_notifications({})
         headers = {
-            'Content-Length': str(len(content)),
-            'Content-MD5': encode_to_base64(get_md5(content.encode('utf-8')))
```

Comment:
Can you test this with 'python 3.5' - the changes related to 'utf-8' were done with this purpose?